protocol/x/clob/e2e/batch_cancel_test.go

```diff
@@ -0,0 +1,283 @@
+package clob_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+// These tests are the same as the e2e tests for single order cancellations.
```

Comment:
can we add a few tests for batch cancelling multiple orders?

Reply:
sep pr